src/unitxt/metrics.py

```diff
@@ -224,134 +253,207 @@ def disable_confidence_interval_calculation(self):
         pass
 
 
+class Aggregator(Artifact):
+    """Aggregates over a list of instances, updating the score/global field of each of them by a dictionary of scores."""
+
+    score_names: List[str] = None
+
+    # non trivial only for grouper_aggregator
+    aggregator_based_score_prefix = ""
```

Comment:
```suggestion
    score_prefix = ""
```
This should be an option to every aggregate (such as mean/ci) then it can add their prefix to every score given from a bottom level aggregator

Reply:
Hi @elronbandel ,  recently, Yoav added this option of `score_prefix` for all **metrics**, that is applicable to instance score as well.
Here we are talking about global scores, and as you said: every **aggregator** has this field.
Currently, for backward compatibility, actual use is made only by `GrouperAggergator`,  but all aggregators have this field, as you see.